docs/draft-release-notes.md

```diff
@@ -1,5 +1,6 @@
 ## Release notes for next release:
 
 ### New feature:
-
+- **Failover service parameters validation is more flexible:** Previously, the validation of the parameters was carried out in a more restrictive way adding rigidity to the admitted values. With this change, the main validation is on the supplier side, and we only check if the field is empty.
```

Comment:
Wouldnt this be a fix rather than new feature?

Reply:
It makes totally sense. I just changed it. Thank you very much @FelisiaM